src/components/ContractsChartPanel.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <app-panel>
+    <template #heading>
+      CONTRACT CALLS
+    </template>
+    <template #header>
+      <contracts-chart-controls
+        class="contracts-chart-panel__chart-controls"
+        @selected="loadContractsStatistics"/>
+    </template>
+
+    <div class="contracts-chart-panel__container">
+      <Line
```

Comment:
With the next chart I will do even more abstraction to reuse code and eliminate code duplication.